import-scripts/preconsume_problematic_samples.sh

```diff
@@ -124,14 +125,18 @@ function consume_hardcoded_samples() {
     rm -f ${PROBLEMATIC_EVENT_CONSUME_IDS_FILEPATH} ${PROBLEMATIC_METADATA_CONSUME_IDS_FILEPATH}
     touch ${PROBLEMATIC_EVENT_CONSUME_IDS_FILEPATH}
     touch ${PROBLEMATIC_METADATA_CONSUME_IDS_FILEPATH}
-    echo "P-0025907-N01-IM6" >> "${PROBLEMATIC_METADATA_CONSUME_IDS_FILEPATH}"
```

Comment:
I mean, know this was here before, but any idea what's this for 😆 Is it not getting caught by our usual script

Reply:
yeah I looked into it, this is a sample we're receiving from the tumor server but it has a normal identifier. Rob added it to this `consume_hardcoded_samples` function until we got more info/a fix from CVR. I reached out to Mihir about it again and he said to continue consuming these and he'll get back to me if there's any change to it